qunetsim/objects/connections/classical_connection.py

```diff
@@ -0,0 +1,44 @@
+from qunetsim.objects.connections.channel_models.classical_model import ClassicalModel
+from qunetsim.objects.connections.connection import Connection
+
+class ClassicalConnection(object):
```

Comment:
Here should extend the `connection` object i.e
```
class ClassicalConnection(Connection)
```
See: https://www.python-course.eu/python3_inheritance.php

Reply:
Done.